polaris.shopify.com/content/patterns/new-features/variants/default.md

```diff
@@ -0,0 +1,46 @@
+---
+hideFromNav: true
+---
+
+## When to highlight new features
+
+Features should have all 3 of the following characteristics before highlighting them to merchants with a New Badge or PIP:
+
+- Shopify wants to drive adoption of this feature because it has high business value
+- It’s creating new and outsized value for merchants (not just an improved way of doing something they are already doing)
+- It’s worth distracting merchants from their current workflow to inform them about a whole new part of Shopify
+
+Things that generally meet this criteria are large and impactful. Examples could be entire new pages in settings or the introduction of an entirely new technology or capability. 
+
```

Comment:
There is a risk that teams have tunnel vision with regards to their new feature, and think that their feature ticks all these boxes. I think we can phrase this even more strongly.